### ISSUE TITLE
Each Video is only playing 1 minute of lesson

### ISSUE THREAD

**Issue by ny2honolulu** [pdeitel/PythonFundamentalsLiveLessons#6]
All videos are playing just a minute of the section the jumping to next.

**Comment by pdeitel** [comment]
Sounds like you were viewing the preview of the videos.

**Comment by ny2honolulu** [comment]
Yes I was but that was a year ago... I've since watched the Full Throttle Live Session and Introductory Artificial Intelligence (AI) ,Big Data and Cloud Case Studies as well...looking forward to new C++ live session as well on Orielly Learning Platform.
